src/main/java/org/galatea/starter/domain/IexHistoricalPrice.java

```diff
@@ -0,0 +1,20 @@
+package org.galatea.starter.domain;
+
+import lombok.Builder;
+import lombok.Data;
+
+import java.math.BigDecimal;
+import java.util.Date;
+
+@Data
+@Builder
+public class IexHistoricalPrice {
+
+  private String symbol;
+  private Date date;
```

Comment:
Use LocalDate here as well

Reply:
@GalateaFarhaad  Had to use java.time.Instant instead of LocalDate as Spring doesn't know how to convert between UNIX timestamps and LocalDate without a timezone being specified. Hopefully this is ok as it is still a Java 8 class and I think it better represents the information returned by the API (which is unzoned)